runtime-modules/storage/src/data_directory.rs

```diff
@@ -89,6 +95,31 @@ decl_error! {
     }
 }
 
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Clone, Encode, Decode, PartialEq, Eq, Debug)]
+pub enum WorkinGroupType {
+    Builders,
```

Comment:
I think for this release lets limit it to `ContentDirectory`, `Builders`, `StorageProviders`

Reply:
Done https://github.com/Joystream/joystream/pull/2099/commits/6f7481efd0170f63797ed930f5bf74e251773ac5